commit 0f829d02468676326a4a57dc498a7a4129a93fab
Author: Rosen Penev <rosenp@gmail.com>
Date:   Thu May 7 17:28:31 2020 -0700

    spdlog: switch to static libraries
    
    Upstream recommends static libraries for smaller size. There is also
    one package that uses spdlog.
    
    Signed-off-by: Rosen Penev <rosenp@gmail.com>

diff --git a/libs/spdlog/Makefile b/libs/spdlog/Makefile
index fb062f4c8..4d0f4e977 100644
--- a/libs/spdlog/Makefile
+++ b/libs/spdlog/Makefile
@@ -7,7 +7,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=spdlog
 PKG_VERSION:=1.5.0
-PKG_RELEASE:=1
+PKG_RELEASE:=2
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://codeload.github.com/gabime/spdlog/tar.gz/v$(PKG_VERSION)?
@@ -29,6 +29,7 @@ define Package/spdlog
   TITLE:=spdlog
   URL:=https://github.com/gabime/spdlog
   DEPENDS:=+libfmt
+  BUILDONLY:=1
 endef
 
 define Package/spdlog/description
@@ -36,7 +37,7 @@ define Package/spdlog/description
 endef
 
 CMAKE_OPTIONS += \
-	-DSPDLOG_BUILD_SHARED=ON \
+	-DSPDLOG_BUILD_SHARED=OFF \
 	-DSPDLOG_BUILD_EXAMPLE=OFF \
 	-DSPDLOG_BUILD_EXAMPLE_HO=OFF \
 	-DSPDLOG_BUILD_TESTS=OFF \
@@ -50,9 +51,4 @@ CMAKE_OPTIONS += \
 
 TARGET_CXXFLAGS += -flto
 
-define Package/spdlog/install
-	$(INSTALL_DIR) $(1)/usr/lib/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libspdlog.so* $(1)/usr/lib/
-endef
-
 $(eval $(call BuildPackage,spdlog))
